nixos/doc/manual/release-notes/rl-2003.xml

```diff
@@ -490,6 +490,52 @@ auth required pam_succeed_if.so uid >= 1000 quiet
      All other display managers in NixOS are configured like this.
     </para>
    </listitem>
+   <listitem>
+     <para>
+       There have been lots of improvements to the Mailman module.  As
+       a result,
+     </para>
+     <itemizedlist>
+       <listitem>
+         <para>
+           The <option>services.mailman.hyperkittyBaseUrl</option>
+           option has been renamed to <xref
+           linkend="opt-services.mailman.hyperkitty.baseUrl"/>.
+         </para>
+       </listitem>
+       <listitem>
+         <para>
+           The <option>services.mailman.hyperkittyApiKey</option>
+           option has been removed.  This is because having an option
+           for the Hyperkitty API key meant that the API key would be
+           stored in the world-readable Nix store, which was a
+           security vulnerability.  A new Hyperkitty will be generated
```

Comment:
A new Hyperkitty will be generated, or a new Hyperkitty API key?

Reply:
Fixed. :)